# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/*/flex/gen.c-fix.patch
# Copyright (C) 1998 - 2004 ROCK Linux Project
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- flex-2.5.31/gen.c	2003-03-30 21:58:44.000000000 +0200
+++ flex-2.5.31/gen.c.old	2003-12-28 00:05:52.000000000 +0100
@@ -1810,11 +1810,8 @@
 	if (yymore_used) {
 		if (!C_plus_plus) {
 			if (yytext_is_array) {
-				if (!reentrant){
-    				indent_puts ("static int yy_more_offset = 0;");
-                }else{
-                    indent_puts ("static int yy_prev_more_offset = 0;");
-                }
+   				indent_puts ("static int yy_more_offset = 0;");
+ 				indent_puts ("static int yy_prev_more_offset = 0;");
 			}
 			else if (!reentrant) {
 				indent_puts
